app/views/quota.html

```diff
@@ -81,16 +81,16 @@ <h3 class="text-center">Memory <small>Limit</small></h3>
                         <!-- Don't show quotas for type `resourcequotas`. They are frequently at limit,
                              which is not something to worry about, and only a cluster admin can create
                              those resources anyway. -->
-                        <tr ng-repeat="(resourceType, specMax) in quota.spec.quota.hard"
+                        <tr ng-repeat="resourceType in orderedTypesByClusterQuota[quota.metadata.name]"
                             ng-if="resourceType !== 'resourcequotas'"
                             ng-class="{
                               warning: isAtLimit(quota, resourceType),
-                              disabled: (quota.status.total.hard[resourceType] || specMax) === '0'
+                              disabled: (quota.status.total.hard[resourceType] || quota.spec.total.hard[resourceType]) === '0'
```

Comment:
if this is replacing what was being set on specMax shouldnt it be quota.spec.quota.hard ?